src/Controller/UI/RemotelyHostedDatasetsController.php

```diff
@@ -155,4 +162,45 @@ private function dispatchLogEvent(Dataset $dataset, string $actor)
             'remotelyhosted_update_log'
         );
     }
+
+    /**
+     * Check to see if the remotely-hosted fields are all populated.
+     *
+     * @param DatasetSubmission $datasetSubmission A DatasetSubmission.
+     *
+     * @return array $errors Containing strings of each unpopulated attribute, or empty array for compliant.
+     */
+    public function remotelyHostedPrerequsiteCheck(DatasetSubmission$datasetSubmission): array
```

Comment:
No space between typehint and parameter?

Reply:
and yet, somehow that worked! hehe oops